packages/integration-tests/src/playbook/contracts/Root.sol

```diff
@@ -0,0 +1,22 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+contract Root {
```

Comment:
What is Root?

Reply:
This is just the name of the contract that we use in the test suite. Its name Root because it must be deployed first and called as a 3rd contract in chain Root <- Middle <- Caller